docs/content/docs/user-docs/install.md

```diff
@@ -15,9 +15,11 @@ The following guide will walk you through the installation of an [ACK service co
 Individual ACK service controllers may be in different maintenance phases and follow separate release cadences. Please check the [project stages][proj-stages] and [maintenance phases][maint-phases] of the ACK service controllers you wish to install, including how controllers are [released and versioned][rel-ver]. Controllers in a preview maintenance phase have at least one container image and Helm chart released to a public repository. 
 
 {{% hint title="Be mindful of maintenance phases" %}}
-Check the [project stage](../../community/releases/#project-stages) and [maintenance phase](../../community/releases/#maintenance-phases) of the ACK service controller you wish to install. Be aware that controllers in a preview maintenance phase may have significant and breaking changes introduced in a future release.
+Check the [project stage][proj-stages] and [maintenance phase][maint-phases] of the ACK service controller you wish to install. Be aware that controllers in a preview maintenance phase may have significant and breaking changes introduced in a future release.
 {{% /hint %}}
 
+[proj-stages]: ../../community/releases/#project-stages
+[maint-phases]: ../../community/releases/#maintenance-phases
```

Comment:
Unfortunately we cannot do this because the markdown within a `{{% hint %}}` block is rendered separately to the rest of the page. Sorry :(

Reply:
When they're taken out, they break the mentions above, so will just add them back in separately to the hint.